Documentation/templates/MigrationGuide.md

```diff
@@ -1,4 +1,27 @@
-# SwiftGen 6.0 Migration Guide
+# SwiftGen 6.2 Templates Migration Guide
+
+### Removed Swift 3 templates
+
+We've removed the bundled `swift3` templates, as we added `swift5` templates. We assume most of you are already developing in a newer version of Swift (4 or 5). If not, you can copy the `swift3` templates from older versions of SwiftGen into your project and point to them using `templatePath`.
+
+### CoreData: entityName is now a property
+
+If you used Core Data templates, the `entityName` is now correctly a `class var` property instead of a `class func`. You will have to remplace calls to `.entityName()` by `.entityName` instead.
+
+### Strings: `%@` now generates an `Any` instead of `String`
+
+When generating constants for strings, if your string contains `%@` placeholders, we previously generated a function with corresponding parameters of type `String`. We now generate these parameters as being of type `Any`, since `%@` corresponds to any class or `NSObject`, not just `String`.
```

Comment:
```suggestion
When generating constants for strings, if your string contains `%@` placeholders, we previously generated a function with corresponding parameters of type `String`. We now correctly generate these parameters as being of type `Any`, since `%@` corresponds to any class or `NSObject`, not just `String`.
```